Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -279,16 +279,6 @@ class SignalAttachment: NSObject {
         return MIMETypeUtil.supportedAudioUTITypes()
     }
 
-    public class var textUTISet: Set<String> {
-        return [
-            kUTTypeText as String,
-            kUTTypePlainText as String,
-            kUTTypeUTF8PlainText as String,
-            kUTTypeUTF16PlainText as String,
-            kUTTypeURL as String
-        ]
-    }
-
     public var isImage: Bool {
         return SignalAttachment.outputImageUTISet.contains(dataUTI)
```

Comment:
We could also use `UTTypeConformsTo()` elsewhere. I'm not sure it would be so valuable.